pinot-common/src/test/java/org/apache/pinot/common/function/scalar/MapFunctionsTest.java

```diff
@@ -0,0 +1,124 @@
+package org.apache.pinot.common.function.scalar;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class MapFunctionsTest {
+
+    @Test
+    public void testRegMapSameTypes()
+            throws JsonProcessingException {
+
+        // CHECKSTYLE:OFF
```

Comment:
(minor) No need to turn off `CHECKSTYLE`, same for other places

Reply:
Addressed